### PR TITLE
Add Gatsby + Contentful articles

### PR DESCRIPTION
**Pull request by martin3walker**
Added to blog posts using Gatsby and Contentful as discussed with [@KyleAMathews](https://github.com/KyleAMathews). 

Here is the original post for [gatsby-contentful-video-tutorials](https://www.contentful.com/blog/2018/02/28/contentful-gatsby-video-tutorials/). Here is the original post for [gatsby-contentful-starter-tutorial](https://www.contentful.com/r/knowledgebase/gatsbyjs-and-contentful-in-five-minutes/).

I'll wait for suggestions on changes to each article. Thanks!

Signed-off-by: Martin Walker

### CODE REVIEW

**Comment by pieh** [comment]
Preview links:
- https://deploy-preview-4913--gatsbyjs.netlify.com/blog/2018-04-05-gatsby-video-tutorials/
- https://deploy-preview-4913--gatsbyjs.netlify.com/blog/2018-04-04-gatsby-contentful-starter-tutorial/

I pushed merge commit to fix build, so just remember to pull changes to your local repo

File: docs/blog/author.yaml [hunk 1]
```diff
@@ -105,3 +105,11 @@
   bio: Full-stack developer, clean code enthusiast, blogger at https://www.vojtechruzicka.com/
   avatar: avatars/vojtech-ruzicka.jpg
   twitter: "@vojtechruzicka"
+- id: Robert Svensson
+  bio: Generally winning
```
Comment (KyleAMathews) [inline docs/blog/author.yaml]: Could you make your bios a bit more descriptive? ;-)

And mention that you work at Contentful.

File: docs/blog/2018-04-05-gatsby-video-tutorials/index.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+title: Gatsby video tutorials a four part video series on Gatsby and Contentful by Khaled Garbaya
+date: 2018-04-05
+author: Carolyn Stransky
+canonicalLink: https://www.contentful.com/blog/2018/02/28/contentful-gatsby-video-tutorials/
+publishedAt: Contentful Blog
+excerpt: Video Tutorials for using Gatsby and Contentful
+---
+
```
Comment (KyleAMathews) [inline docs/blog/2018-04-05-gatsby-video-tutorials/index.md]: Could you add at the top in italics that the blog posts that they were originally posted on the Contentful blog?

File: docs/blog/author.yaml (hunk 1 above)
Comment (martin3walker) [reply docs/blog/author.yaml]: Yes, sorry! I meant to change that, and completely forgot.

File: docs/blog/2018-04-05-gatsby-video-tutorials/index.md (hunk 2 above)
Comment (martin3walker) [reply docs/blog/2018-04-05-gatsby-video-tutorials/index.md]: When you say at the top, do you mean in the title?

**Comment by m-allanson** [comment]
Thanks @martin3walker! Along with @KyleAMathews comments above, I have one more request: 

Gatsby recently added a requirement that contributors sign-off commits. Could you do `git rebase --signoff` and force push the amended commit?

Check out the new [Developer Certificate of Origin
 docs](https://www.gatsbyjs.org/docs/how-to-contribute/#developer-certificate-of-origin) or comment here if you've any questions.

File: docs/blog/2018-04-05-gatsby-video-tutorials/index.md (hunk 2 above)
Comment (martin3walker) [reply docs/blog/2018-04-05-gatsby-video-tutorials/index.md]: Hey I made the changes in line 10

**Comment by KyleAMathews** [comment]
Thanks! Excited to get these out there!

**Comment by jlengstorf** [comment]
Hiya @martin3walker! 👋

This is definitely late, but on behalf of the entire Gatsby community, I wanted to say thank you for being here.

Gatsby is built by awesome people like you. Let us say “thanks” in two ways:

1.  **We’d like to send you some Gatsby swag.** As a token of our appreciation, you can go to the [Gatsby Swag Store][store] and log in with your GitHub account to get a coupon code good for one free piece of swag. (We’ve got t-shirts and hats, plus some socks that are really razzing our berries right now.)
2.  **If you’re not already part of it, we just invited you to join the Gatsby organization on GitHub.** This will add you to our team of maintainers. You’ll receive an email shortly asking you to confirm. By joining the team, you’ll be able to label issues, review pull requests, and merge approved pull requests.

If you have questions, please don’t hesitate to reach out to us: tweet at [@gatsbyjs][twitter] and we’ll come a-runnin’.

Thanks again! 💪💜

[store]: https://store.gatsbyjs.org
[twitter]: https://twitter.com/gatsbyjs

**Pull request merged**